From 400104fa5f45f4464311ec499716786ab1f4a5df Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Sun, 21 Jan 2024 13:05:17 +0100
Subject: [PATCH] Include the xterm-256color terminfo entry in
 di-utils-terminfo

This is a prerequisite for switching from the old unmaintained vte
package to vte2.91, as the latter sets TERM to xterm-256color by
default.

Closes: #1061121
---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index c7278c7..ed9690c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -17,7 +17,7 @@ export DEB_CFLAGS_MAINT_APPEND := -Wall -W -Os -fomit-frame-pointer
 CFLAGS := $(shell dpkg-buildflags --get CPPFLAGS; dpkg-buildflags --get CFLAGS)
 LDFLAGS := $(shell dpkg-buildflags --get LDFLAGS)

-TERMNAMES = a/ansi d/dumb s/screen x/xterm
+TERMNAMES = a/ansi d/dumb s/screen x/xterm x/xterm-256color

 ifeq ($(DEB_HOST_ARCH_OS),linux)
 TERMNAMES += l/linux
--
2.43.0

